src/app/transcation-listing/transcation-listing.component.css

```diff
@@ -0,0 +1,167 @@
+
+    .Rectangle-4 {
+    position: absolute;
+    top: 50px;
+    /* bottom: 755px; */
+    bottom: 818px;
+    right: 235px;
+    width: 834px;
+    /* height: 59px; */
+    box-shadow: 0 1px 0 0 #d1d1d1;
+    background-color: #ffffff;
+    padding: 0px;
+    font-size: 14px;
+    color: #666666;
+    overflow-y: scroll;
+    font-weight: 500;
+  }
+  .nedbank-green {
+    width: 1440px;
+    height: 80px;
+    background-color: #006341;
+  }
+  .Down {
+    width: 24px;
+    height: 24px;
+    object-fit: contain;
+    size: 24px;
+  }
+
+  .table-scroll{
+    /* display: unset; */
+    position: absolute;
+    height: 400px;
+    width: 100%;
+    overflow: scroll;
+  }
+
+  
+  .view {
+    left: 300px;
+    right: 330px;
+    width: 204px;
+    height: 18px;
+    font-family: MarkPro;
+    font-size: 14px;
+    font-weight: normal;
```

Comment:
I believe no need to define value normal by default it takes normal, please check.